src/native/libs/Common/pal_utilities.h

```diff
@@ -130,3 +136,24 @@ inline static int32_t SizeTToInt32(size_t value)
     assert(value <= INT_MAX);
     return (int32_t)value;
 }
+
+/**
```

Comment:
If you agree with the other feedback, I would undo this change and just use the Android specific libc name in the Android specific workaround.